examples/sglang/configs/embedding.yaml

```
@@ -0,0 +1,16 @@
+Frontend:
+  served_model_name: e5
+  endpoint: SGLangEmbeddingWorker.generate
+  port: 8000
+SGLangEmbeddingWorker:
+  model-path: intfloat/e5-base-v2
+  served-model-name: e5
+  is-embedding: true
+  tp: 1
+  trust-remote-code: true
+  is-embedding: true
```

YAMLlint (1.37.1)

[error] duplication of key "is-embedding" in mapping

(key-duplicates)